### PR TITLE
Add comments

### PR DESCRIPTION
**Pull request by Lunalo**

### CODE REVIEW

File: instat/frmEditor.vb [hunk 1]
```diff
@@ -676,6 +676,7 @@ Public Class frmEditor
     End Sub
 
     Private Sub AddCommentToolStripMenuItem_Click(sender As Object, e As EventArgs) Handles AddComment.Click
+        dlgAddComment.ucrReceiverColumn.txtReceiverSingle.Text = SelectedColumnsAsArray()(0)
```
Comment (dannyparsons) [inline instat/frmEditor.vb]: You should never set the Text of the textbox directly. See how this is done for other dialog like Convert to Date.

File: instat/dlgAddComment.vb [hunk 2]
```diff
@@ -24,14 +27,29 @@ Public Class dlgAddComment
...
 
+
+    Public Sub SetCurrentColumn(strColumn As String, strDataFrame As String)
+        strSelectedColumn = strColumn
+        strSelectedDataFrame = strDataFrame
+        bUseSelectedColumn = True
+    End Sub
+
+    Private Sub SetDefaultColumn()
+        ucrSelectorAddComment.ucrAvailableDataFrames.cboAvailableDataFrames.SelectedItem = strSelectedDataFrame
```
Comment (dannyparsons) [inline instat/dlgAddComment.vb]: There is a Set Data Frame method to use instead of this.

**Comment by Lunalo** [comment]
This is ready

**Comment by dannyparsons** [comment]
Great, can someone test?

**Comment by Lunalo** [comment]
@dannyparsons 

I would request this to be merged. I wanted to update this branch but there were conflicting files.

**Comment by maxwellfundi** [comment]
I have been testing this and the dialog does not run any function yet.  @dannyparsons please merge this so that the  code can be updated by @Lunalo

**Pull request merged**